src/cli.zig

```diff
@@ -139,14 +145,19 @@ pub fn parse_args(allocator: std.mem.Allocator) !Command {
         const arg = try parsed_arg;
         try args_allocated.append(arg);
 
+        // TODO: inline-for this?
         if (mem.startsWith(u8, arg, "--cluster")) {
             cluster = parse_flag("--cluster", arg);
         } else if (mem.startsWith(u8, arg, "--replica")) {
             replica = parse_flag("--replica", arg);
         } else if (mem.startsWith(u8, arg, "--addresses")) {
             addresses = parse_flag("--addresses", arg);
-        } else if (mem.startsWith(u8, arg, "--memory")) {
-            memory = parse_flag("--memory", arg);
+        } else if (mem.startsWith(u8, arg, "--cache-accounts")) {
+            cache_accounts = parse_flag("--cache-accounts", arg);
+        } else if (mem.startsWith(u8, arg, "--cache-transfers")) {
+            cache_transfers = parse_flag("--cache-transfers", arg);
+        } else if (mem.startsWith(u8, arg, "--cache-transfers-posted")) {
+            cache_transfers_posted = parse_flag("--cache-transfers-posted", arg);
```

Comment:
This is getting too repetetive. Do we want to DRY this out? If we do, I'll probably send a separate PR for that.

Reply:
Yeah. Especially since we have other CLIs that need to parse arguments too — right now it is all ad-hoc.

Reply:
https://github.com/tigerbeetledb/tigerbeetle/pull/282